src/config.test.ts

```diff
@@ -0,0 +1,137 @@
+import { safeLoad } from "js-yaml";
+import {
+  migrateSkycliConfig,
+  updateUser,
+  getUser,
+  deleteUser,
+  createSkycliConfig,
+} from "./config";
+
+const fixture = `
+cluster:
+  default:
+    endpoint: 'http://controller.skygear.localhost:8081'
+    api_key: api_key
+    env: enterprise
+context:
+  default:
+    cluster: default
+    user: default
+current_context: default
+user:
+  default:
+    user:
+      id: user_id
+      created_at: '2019-12-06T08:53:10.413Z'
+      last_login_at: '2020-01-09T06:51:55.549Z'
+      is_verified: true
+      is_disabled: false
+      metadata: {}
+    identity:
+      id: identity_id
+      type: password
+      login_id_key: email
+      login_id: user@example.com
+      claims:
+        email: user@example.com
+    access_token: access_token
+    refresh_token: refresh_token
+    session_id: session_id
+`;
+
+const expected = `
+version: v1
```

Comment:
I think for consistency using `api_version` is better.

Reply:
So the field name is the same but it is entirely different thing than the API version we have in Controller. Is it OK? @carmenlau

Reply:
I think it is okay to use name `api_version` with different value, skycli config should have its own version.